### PR TITLE
added chdir call after successful chroot

### PR DESCRIPTION
**Pull request by jvymazal**
attempt to solve #1419

### CODE REVIEW

File: tools/rsyslogd.c [hunk 1]
```diff
@@ -1350,7 +1350,7 @@ initAll(int argc, char **argv)
 				perror("chroot");
 				exit(1);
 			}
-            if(chdir("\\") != 0) {
```
Comment (rgerhards) [inline tools/rsyslogd.c]: please fix indenting to comply to the style used in the rest of the switch

**Comment by rgerhards** [comment]
It must be noted that the -T option is a testing option, and not meant for secure operations (as per the comment). So this is not an option for general use. I would suspect that the change could even cause some scenarios to fail, but I would accept that until I get feedback from tests that this poses a real issue.

**Comment by rgerhards** [comment]
sorry, I forgot: could you squash the changes, as it really is just one change with two fixes to clean up problems in the change itself. If you do not know how to do this, I can do it for you (assuming you have given appropriate permissions).

**Comment by jvymazal** [comment]
Ok, I squashed the commits, even if -T is presently testing option it would be nice to have it there (for example if it would become security option one day). Thanks.

**Pull request merged**